### DISCUSSION TITLE
Is there a minimum Em size for bitmap fonts?

### DISCUSSION THREAD

**Discussion by Kiaazad** [fontforge/fontforge#5207] [General]
I'm trying to create a font that's 8 pixels tall, but when I export it into .fnt format, the sizes change on me.
![image](https://user-images.githubusercontent.com/33455007/220340537-e5b80dbc-8da2-435e-b89d-e4d35b97d949.png)
I wonder if there are limitations on how small I can go, because I had the `Self Intersecting` problem when I've tried to make a vector font with both Em sizes 8 and 16, but only the Em size 8 wouldn't open in windows font viewer, giving me this error.
![image](https://user-images.githubusercontent.com/33455007/220342496-f5433727-1a8c-4c27-9e5c-5067bf2d4704.png)

**Comment by ctrlcctrlv** [comment]
Slow down. Vector font or bitmap font? Your title says one and your question the other.

**Reply by Kiaazad** [reply]
Both, the first screenshot is from bitmap font when I generate one with Em of 8 and open the font in fontforge again., the second what happens when I generate a vector font with Em of 8 and try to open it in windows font viewer.

**Reply by ctrlcctrlv** [reply]
You are unchecking Scale Outlines?

**Reply by Kiaazad** [reply]
No, I only changed `Em size` and the `Ascent, Decent` settings.
